src/js/components/app.js

```diff
@@ -18,6 +19,19 @@ import { getOnboardingComponentFor } from '../utils/onboardingmanager';
 const timeout = 900000; // 15 minutes idle time
 
 class AppRoot extends React.PureComponent {
+  componentDidMount() {
+    this.props.history.listen(location => {
+      let page = location.pathname || '';
+      ReactGA.set({ page });
+      // unless we're on a help page the path will get cut off to the rough region, to ensure we're not sending any sensitive device/ group/ deployment names etc.
+      if (!location.pathname.startsWith('/help')) {
+        const path = location.pathname.split('/');
+        page = path.length > 1 ? path[1] : path[0];
+      }
```

Comment:
I'm not 100% sure this is a good idea: we are losing details about what the user is doing and how he transitions from one page to another, without the full path, don't you think so? If you are concerned about disclosing info, do you think it is possible to track the page view using the name of the route (or the component used to render the route) instead?

Reply:
I tried to make it a bit more detailed, but wanted to stay away from adding the tracking in the different components themselves... the only other place with a more unified access to the components & route would be the router, however the component access there is limited to the top level containers and doesn't really provide additional insight, compared to just the rough area... dialogs are a bit outside of this flow though... hmm I'll try to find a way to hook into their lifecycle...